src/getHumanAge.test.js

```diff
@@ -1,10 +1,36 @@
-'use strict';
+"use strict";
 
-describe('getHumanAge', () => {
-  const { getHumanAge } = require('./getHumanAge');
+describe("getHumanAge", () => {
+  const { getHumanAge } = require("./getHumanAge");
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test("should be declared", () => {
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  test("should return correct values for valid inputs", () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+    expect(getHumanAge(23, 24)).toEqual([1, 2]);
+    expect(getHumanAge(100, 100)).toEqual([21, 17]);
+  });
+
+  test("should have expected values with rounds", () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+    expect(getHumanAge(16, 16)).toEqual([1, 1]);
+    expect(getHumanAge(16, 24)).toEqual([1, 2]);
+    expect(getHumanAge(28, 28)).toEqual([3, 2]);
+  });
+
+  test("should handle invalid and edge inputs", () => {
+    expect(getHumanAge(-5, -10)).toEqual([0, 0]);
+    expect(getHumanAge(14.5, 14.7)).toEqual([0, 0]);
+    expect(Number.isNaN(getHumanAge(NaN, 14)[0])).toBe(true);
+    expect(Number.isNaN(getHumanAge(14, NaN)[1])).toBe(true);
+    expect(getHumanAge(NaN, undefined)).toEqual([NaN, NaN]);
+  });
+
+  test("should correctly handle invalid types", () => {
+    expect(getHumanAge("15", 15)).toEqual([1, 1]);
```

Comment:
The test case `expect(getHumanAge("15", 15)).toEqual([1, 1]);` assumes that a string input should be converted to a number. However, this behavior is not defined in the `getHumanAge` function, and typically, string inputs should result in an error or `NaN`. Consider updating the function to handle string inputs explicitly or adjust the test case to expect `NaN`.